.eslintrc.json

```diff
@@ -31,26 +31,12 @@
     "no-inner-declarations": "off",
     "valid-jsdoc": "off",
     "require-jsdoc": "off",
-    "quotes": ["error", "backtick"],
     "consistent-return": ["error"],
     "arrow-body-style": [
       "error",
       "as-needed",
       { "requireReturnForObjectLiteral": true }
     ],
-    "jsx-quotes": ["error", "prefer-double"],
-    "semi": ["error", "never"],
-    "object-curly-spacing": ["error", "always"],
-    "comma-dangle": [
-      "error",
-      {
-        "arrays": "always-multiline",
-        "objects": "always-multiline",
-        "imports": "always-multiline",
-        "exports": "always-multiline",
-        "functions": "ignore"
-      }
-    ],
```

Comment:
Remove all style related rules from ESLint as Prettier is taking care of them

Reply:
will commit hooks run when someone is making changes via github edit UI / github merge conflicts resolver view? if not it would be beneficial to actually keep those I think

Reply:
We will have the CI tests for us if the linting is failing